plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/OpenshiftJMXConnectionProvider.java

```diff
@@ -50,11 +53,11 @@ public String getId() {
 	@Override
 	protected IConnectionWrapper createConnection(IServer server) {
 		IConnection openshiftCon = OpenShiftServerUtils.getConnection(server);
-		IService service = OpenShiftServerUtils.getService(server);
+		IResource resource = OpenShiftServerUtils.getResource(server);
```

Comment:
yes and also if the connection is null. Will be reported to the user probably. And this was an old code but will fix it